Encapsulation Exercises/temperature-converter/src/TemperatureConverter.java

```diff
@@ -0,0 +1,19 @@
+public class TemperatureConverter {
+    public static void main(String[] args) {
+        // Example usage of Temperature class through TemperatureConverter
+        Temperature temperature = new Temperature(32.0);
```

Comment:
create a text based interface instead.
ask an input value and current temperature unit and convert to other temperature unit as desired.

Reply:
work on this